eras/allegra/impl/src/Cardano/Ledger/Allegra/TxBody.hs

```diff
@@ -84,7 +84,7 @@ import GHC.Generics (Generic)
 import Lens.Micro
 import NoThunks.Class (NoThunks (..))
 
-class ShelleyEraTxBody era => AllegraEraTxBody era where
+class EraTxBody era => AllegraEraTxBody era where
```

Comment:
I suppose removing `ShelleyEraTxBody`  constraint was necessary so that Conway can still have an instance for this, correct?

Reply:
Correct.